blockchain/chain.go

```diff
@@ -551,10 +601,10 @@ func (b *BlockChain) loadBlockNode(dbTx database.Tx, hash *chainhash.Hash) (*blo
 }
 
 // findNode finds the node scaling backwards from best chain or return an
-// error.
+// error.  If searchDepth equal zero there is no searchDepth.
 //
 // This function MUST be called with the chain state lock held (for writes).
-func (b *BlockChain) findNode(nodeHash *chainhash.Hash) (*blockNode, error) {
+func (b *BlockChain) findNode(nodeHash *chainhash.Hash, searchDepth int) (*blockNode, error) {
```

Comment:
Because one may be looking for the stats of block 12 while the chain is at 90000.